delete-operator.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -x
+
+if [[ -z "${TNF_PARTNER_NAMESPACE}" ]]; then
+    export TNF_PARTNER_NAMESPACE="tnf"
+fi
+
+# Cleanup previous deployment if present
+operator-sdk cleanup nginx-operator -n $TNF_PARTNER_NAMESPACE
```

Comment:
this will keep the OLM and SDK installed right ?
I'm just wondering If I run clean; then install again, will it work without issue in local machine ?
In github CI, it's not an issue, since it's new machine everytime.

Reply:
yes, It should work, I tested this scenario